README.md

```diff
@@ -300,6 +301,59 @@ struct Product {
 let totalCents = products.reduce(0) { $0.amount + $1.amount }
 ```
 
+## Nanolibraries
+
+The `Tagged` library also comes with a few nanolibraries for handling common types in a type safe way.
+
+### `TaggedTime`
```

Comment:
Wonder if we want "safe" conversion to `Date`, since that's a common thing to do with seconds/milliseconds and prone to errors?

Reply:
good idea, I did that but don't know if I did it correctly. check it out 257eed1